index.html

```diff
@@ -237,17 +248,19 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
-    <section class="contact">
-        <div class="container">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-        </div>
-        <div class="container" id="contactform">
-            <form action="https://formspree.io/f/xnqydbrv" method="POST">
-                <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required><br>
-                <input type="email" id="email" name="email" placeholder="Email address" required><br>
-                <textarea id="comments" name="comment" rows="9" placeholder="Enter text here" maxlength="500" required></textarea><br>               
-                <input class="button" type="submit" value="Get in touch">
-            </form>
+    <section class="contactsection">
+        <div class="contact">
+            <div class="container">
+                <h2 id="contactformtext">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </div>
+            <div class="container" id="contactform">
+                <form action="https://formspree.io/f/xnqydbrv" method="POST">
+                    <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required><br>
+                    <input type="email" id="email" name="email" placeholder="Email address" required><br>
+                    <textarea id="comments" name="comment" rows="9" placeholder="Enter text here" maxlength="500" required></textarea><br>               
+                    <input class="button" id="formbutton" type="submit" value="Get in touch">
+                </form>
+            </div>
```

Comment:
Very well done also in this section , your contact form is perfectly scaling with screen size 💪 Only one thing to mention is the vertical spacing with other sections 

**figma design:**

![figma contact me desktop](https://user-images.githubusercontent.com/59707859/220773835-0f6a9d03-c07f-4b42-84e0-bc7a25217232.PNG)

**your desing:**

![student contact me desktop](https://user-images.githubusercontent.com/59707859/220773948-6a81cf72-00f2-4265-b8b0-8a7c73c56a54.PNG)